src/legacy/ui/public/vis/__tests__/map/ems_client_util.js

```diff
@@ -18,7 +18,7 @@
  */
 
 
-import { EMSClient } from '../../../../../core_plugins/tile_map/common/ems_client';
+import { EMSClient } from '@elastic/ems-client';
```

Comment:
yes. Let's just remove it. There was a specific reason for this to include it.

- the ems-client as in core Kibana, but had to be imported into Maps (both in the app, but also in the unit test code).
- for the unit tests, we needed to shim the implementations so the client would _not_ connect to the live EMS, but provide dummy responses.

This wiring was the way to make that work. But with this extracted to a standolone package, some of these issues no longer apply.

What should be preserved is tests for the correct instantiation of the client based on some app-params (the `meta.test.js`, proxying/not-proxiing basically) file. Maybe this can be rewritten to _not_ have to rely on the ems-client.

Reply:
After in person discussion, that `meta.test.js` file doesn't really seem to test anything meaningful. It just tests if the boolean config of proxyElasticMapsServiceInMaps propagates correctly for the construction of the EMS-client.